docs/sdkx-unreal/getting-started.mdx

```diff
@@ -31,9 +31,13 @@ Guide to integrating the Unreal Engine plugin for the Helpshift SDK X.
 
 ## Requirements {#version-requirements}
 
-- Unreal 4.18 and above.
-- Supported Android OS versions: 16 and above
-- Supported iOS versions: iOS 15, 14, 13, 12 and 11
+- Unreal 4.26 and above.
+- Supported Android OS versions:
+  - SDK X is functional only above API Level 24 i.e Android OS 7 Nougat and above.
+  - The minSDKVersion in code is kept as API Level 16 to allow importing the SDK and compiling in apps that allow installation till API Level 16 i.e Android 4.4 Jelly Bean.
+  - If your app is installed on a device below API level 24 (i.e below Android OS 7 Nougat) then you can choose to have a different medium for providing support. For example, Email, Web etc. We log an UnsupportedOSVersionException if the SDK is initialized below API Level 24.
```

Comment:
UnsupportedOSVersionException
Not sure if we throw this exception in Unreal C++ layer.
Please check

Reply:
we don't throw this Exception but we log it thats why i mention  **_We log an UnsupportedOSVersionException_**